.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -1,119 +1,146 @@
-name: Bug Report
-description: Helps us improve our product!
-labels: ["Needs triage", "[Type] Bug"]
-type: 'Bug'
+name: IPSM Bug Report v2
```

I think "IPSM" and "v2" will mean very little to most people, including a12s. I think just "Bug Report" would be enough.

```suggestion
name: Bug Report
```

---

Placeholder. Happy to update it to the same as the one this replaces. Would you like me to commit these suggestions, or will you take care of that? I'm unfamiliar with the process.

---

Done in a43c98ff65eb0f74d6f1aedb5d6235a36c6a8da9